.travis.yml

```diff
@@ -1,6 +1,12 @@
-sudo: required
+sudo: false
+dist: trusty
 language: node_js
 node_js:
-  - 8
+  - 'lts/*'
```

Comment:
Looks like travis should automatically pick up the nvmrc https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions-using-nvmrc